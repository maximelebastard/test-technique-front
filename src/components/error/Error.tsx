import React from "react";
import styles from "./Error.module.scss";

const Error: React.FC = () => {
  return (
    <div className={styles.Error}>
      We&apos;re sorry, an error occured. Please check your internet connection
      and try again.
    </div>
  );
};

export default Error;
