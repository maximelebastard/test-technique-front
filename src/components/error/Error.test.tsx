import React from "react";
import { shallow, render } from "enzyme";
import Error from "./Error";

it("renders without crashing", () => {
  shallow(<Error />);
});

it("matches the snapshot", () => {
  expect(render(<Error />).text()).toMatchSnapshot();
});
