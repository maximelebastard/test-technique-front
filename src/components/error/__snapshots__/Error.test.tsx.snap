// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `"We're sorry, an error occured. Please check your internet connection and try again."`;
